client/src/components/App.jsx

```diff
@@ -237,11 +256,45 @@ class App extends React.Component {
         const stockSelected = res.data;
         this.setState({
           stockSelected
+        }, () => {
+          this.forceUpdate();
```

Comment:
Why do we need to force an update??

Reply:
Good catch!! It was the only way to render a stock detail before, when I had the helper function nested and was using app level state. I ended up pulling the function and the state up now though and am just rendering with props- I just tested and it will work without it